### ISSUE TITLE
Angury not working with debugging web extension apps

### ISSUE THREAD

**Issue by kspearrin** [rangle/augury#1286]
Augury version (required): 1.16.0
Angular version (required): 5.2.0
Date: 
OS: Windows 10

Demo test application:
https://github.com/bitwarden/browser

Description of issue:
Angury does not seem to work with Angular apps running in a (Chrome) web extension popup.

Steps to reproduce:

1. Create a popup browser action that is an Angular app.
2. Debug with dev tools.
3. Switch to Angury tab. None of the Angury tabs shows any information about the running app.

![image](https://user-images.githubusercontent.com/1190944/38756461-9e109b46-3f37-11e8-873e-a514c43fab75.png)

Additional details:

- App is packaged with webpack, running with JIT compilation, dev mode.
- Chrome extensions run under the `chrome-extension://` protocol so maybe that has something to do with it?

P.S. Sorry if this has been posted before. I did some search but wasn't want to locate anything.

**Comment by andrewthauer** [comment]
@kspearrin - Do you mind trying the latest version of Augury. We've made quite a few stabilization fixes since the release posted.

**Comment by kspearrin** [comment]
@andrewthauer Yes, I just tested with Augury 1.19.1 and this is still an issue.

**Comment by andrewthauer** [comment]
@kspearrin - Without additional details as to the errors it's hard to say. The use case of debugging another Angular devtool with Augury seems a bit low. However, in theory this should work. But it looks like you have a fairly customized build setup, etc. so it could be a number of things going on. Happy to look at it when we get a chance.

In the meantime it might help if you where able to inspect the augury devtool window itself and see if there are any console errors showing up. You should be able to right click on the augury UI and click "inspect". This will pop up another inspector window which hopefully has some additional details.

**Comment by andrewthauer** [comment]
Yeah, that's odd. I would have expected to see something. Did you try reloading after you inspected? It's possible any actual errors only occur when your angular app first bootstraps.

**Comment by kspearrin** [comment]
@andrewthauer Reloading my app after having the dev tools open didn't seem to change anything.
